code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -222,56 +248,69 @@
 	id = "patron"
 	description = "Tequila with silver in it, a favorite of alcoholic women in the club scene."
 	color = "#585840" // rgb: 88, 88, 64
-	boozepwr = 45
+	boozepwr = 60
 
 /datum/reagent/consumable/ethanol/gintonic
 	name = "Gin and Tonic"
 	id = "gintonic"
 	description = "An all time classic, mild cocktail."
 	color = "#664300" // rgb: 102, 67, 0
-	boozepwr = 55
+	boozepwr = 25
 
 /datum/reagent/consumable/ethanol/cuba_libre
 	name = "Cuba Libre"
 	id = "cubalibre"
-	description = "Rum, mixed with cola. Viva la revolution."
+	description = "Rum, mixed with cola. Viva la revolucion."
 	color = "#3E1B00" // rgb: 62, 27, 0
-	boozepwr = 45
+	boozepwr = 50
+
+/datum/reagent/consumable/ethanol/cuba_libre/on_mob_life(mob/living/M)
+	if(M.mind && M.mind.special_role in list("Revolutionary", "Head Revolutionary")) //Cuba Libre, the traditional drink of revolutions! Heals revolutionaries.
+		M.adjustBruteLoss(-1)
+		M.adjustFireLoss(-1)
+		M.adjustToxLoss(-1)
+		M.adjustOxyLoss(-5)
```

Comment:
`.` is what the current proc returns.
`..()` is calling the parent proc.
We want the proc to return 1 when any of the adjustxxxx() is called, so if there's one in the current proc we do `. = 1`, but we also have to call `..()` because we want to run the code in the parent proc. This parent proc itself can have an adjustxxx(), so to cover all cases we need `return ..() || .` 
Sometimes the current proc doesn't have an adjustxxx() but the parent will have one, so just doing `..()` won't make our proc return the needed 1.